### DISCUSSION TITLE
cityjson -> dotbim conversion

### DISCUSSION THREAD

**Discussion by paireks** [cityjson/specs#173] [General]
Hello guys!

I wanted to reach out to you if you'd be interested to help me out figuring out such conversion :) 
Generally all I need to make it happen is simple access to all your objects' meshes. If I could simply get your mesh results (via C# - preffered choice, or via Python) then I could easily convert them to .bim meshes. It would be even better if I could get your meshes + meta-data attached. But the question for you is if there is some simple way to do it? :)

Best regards,
Wojciech :)

**Comment by hugoledoux** [comment]
check the code of [cjio](https://github.com/cityjson/cjio) where we parse files and extract geometries, similar to what you want to do.

**Reply by hugoledoux** [reply]
and this would help with the different geometry types: https://www.cityjson.org/dev/geom-arrays/
